worker/src/DepUsrSCTP.cpp

```diff
@@ -249,7 +252,21 @@ void DepUsrSCTP::Checker::OnTimer(TimerHandle* /*timer*/)
 	auto nowMs          = DepLibUV::GetTimeMs();
 	const int elapsedMs = this->lastCalledAtMs ? static_cast<int>(nowMs - this->lastCalledAtMs) : 0;
 
+#ifdef MS_LIBURING_SUPPORTED
+	// Activate liburing usage.
+	// 'usrsctp_handle_timers()' will synchronously call the send/recv
+	// callbacks for the pending data. If there are multiple messages to be
+	// sent over the network then we will send those messages within a single
+	// system call.
+	DepLibUring::SetActive();
+#endif
+
 	usrsctp_handle_timers(elapsedMs);
 
+#ifdef MS_LIBURING_SUPPORTED
+	// Submit all prepared submission entries.
+	DepLibUring::Submit();
```

Comment:
Why here? Submit should be called when messages are about to be dents here we just called to handle timers.

Reply:
As indicated in the comment few lines above, the call to `usrsctp_handle_timers()` synchronously call the send/recv static callbacks that we provided to `usrsctp_socket` in `SctpAssociation` for all the send/recv enqueued tasks in the usrsctp sockets, so it's very probable that multiple messages will be sent over the network between the new `DepLibUring::SetActive()` and `DepLibUring::Submit()`. Hence, it's important that when those messages are about to be sent `DepLibUring` is active so they are prepared in the ring or order to be later sent at once upon calling `DepLibUring::Submit()`